### ISSUE TITLE
space before the heading

### ISSUE THREAD

**Issue by valantien** [Wittionary/mammoth-obsidian-theme#15]
First, thanks for the amazing theme.

Is there a way to remove/disable the space before the #headigns ?  
The misalignment between the header and the rest of paragraphs, drives me crazy. lol

**Comment by Wittionary** [comment]
Hey, thanks! I'm glad you're enjoying it ☺

Currently there is not a way to do that. If I figure out how to implement some sort of toggle, I'd like to try that route.

I'll reconsider the indentation depths between the different headers. I'm wondering if I made a choice that was aesthetic before it was functional. **The original idea behind it was to be able to quickly distinguish the hierarchy by both color and depth.** If I remove the depth, then we're left with just the color which relies on the user memorizing the order of those.

For you, is what's driving you crazy that the headers don't seem to be rooted and are sort of floating away from the left margin? Or, if the paragraph/block below was aligned with the header above it, would that solve the irritation?

Just to set the expectation, I reserve the right to keep it as is, but I definitely want to hear back from those who are using it to see what can be improved and how I may be able to solve the hierarchy-recognition problem another way!

**Comment by valantien** [comment]
Hi Wittinary thank you for your reply.

To give you a feedback, yes what bothers me visually is the space between the margin and the header.
As I work as a book editor these details for me are more glaring.

But ok I understand the logic behind your decision, just IMHO if you want to visually emphasize the headers, maybe it's better then increase this space between margin and header, because as it stands, it looks more  like an unintentional error.

Please this is just an opinion I could be wrong.

**Comment by valantien** [comment]
I think I found the source of the problem.

As I used the "Fold Indent" option turned off in options/editor, and because of that, the indentation arrows don't appear for me before the headers, making it even bigger, the space between the margin and the header.

![Screen Shot 2021-07-22 at 11 39 54](https://user-images.githubusercontent.com/65976883/126658433-0202b686-f6e8-4494-9625-8637995ea9dd.png)

I turned on the Fold Indent option.
